		} catch (IOException err) {
		} finally {
			try {
				cIn.close();
			} catch (IOException e) {
			}
